mk/lib.mk

```diff
@@ -62,13 +63,17 @@ ifeq ($(CFG_ULIBS_SHARED),y)
 ifeq ($(sm)-$(CFG_TA_BTI),ta_arm64-y)
 lib-ldflags$(lib-shlibfile) += $$(call ld-option,-z force-bti) --fatal-warnings
 endif
-$(lib-shlibfile): $(objs) $(lib-needed-so-files)
+ifneq (,$(wildcard $(lib-verscript)))
```

Comment:
I'm not so sure about the `wildcard`, that could hide an invalid value in `lib-verscript`.